src/ScottPlot4/ScottPlot/NumeriConversion.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+
+namespace ScottPlot
+{
+    public static class NumeriConversion
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static double GenericToDouble<T>(T value)
+        {
+            return value switch
+            {
+#if NET
```

Comment:
Just for my ignorance, `NET` corresponds to .NET 5 or greater?

It may make sense to replace it with `NET5_0_OR_GREATER`, because to most people .NET could reasonably encompass .NET Framework, Core, and 5+, and potentially even Mono.

Again, this is a matter of taste, anyone who has to modify this code will figure it out, it just saves them a few swearwords at Microsoft's naming scheme.

Reply:
Switched to NETCOREAPP